server/src/main/java/io/littlehorse/common/model/getable/core/wfrun/ThreadRunModel.java

```diff
@@ -620,6 +621,40 @@ public void advanceFrom(NodeModel curNode) {
         }
     }
 
+    public void advanceFrom(VariableValueModel output, NodeModel curNode) {
```

Comment:
I just copied the `advanceFrom` twice, so now there are two `advanceFrom` one that mutates and one that doesn't. I should find a better way of doing this without copying the exact same code twice

Reply:
Are there clients of both `advanceFrom()` methods, and if so, should we migrate all to use the one that mutates?